catalog/spatial/spatial-app/src/main/resources/features.xml

```diff
@@ -58,6 +58,13 @@
         </bundle>
     </feature>
 
+    <feature name="spatial-wfs-v1_1_0" install="auto" version="${project.version}"
+             description="Web Feature Service (WFS) v1.1.0 Source.">
+        <feature prerequisite="true">spatial-app</feature>
+        <feature prerequisite="true">spatial-wfs-core</feature>
+        <bundle>mvn:org.codice.ddf.spatial/spatial-wfs-v1_1_0-source/${project.version}</bundle>
```

Comment:
❗️ spatial-wfs-v1_1_0-converter should be included here since it is a bundle

Reply:
changed